.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
I'd rather not have systemspecific stuff in the project-specific gitignore file. That is something for a users global gitignore file

Reply:
I removed the commit